cirq/circuits/circuit_test.py

```diff
@@ -3817,3 +3817,29 @@ def test_zip():
             cirq.Circuit(cirq.X(a), cirq.CNOT(a, b)),
             cirq.Circuit(cirq.X(b), cirq.Z(b)),
         )
+
+
+def test_repr_html_escaping():
+
+    class TestGate(cirq.Gate):
+
+        def num_qubits(self):
+            self.n_qubits = 2
+            return self.n_qubits
```

Comment:
don't be afraid of hard-coding in tests. 

```
def num_qubits(self):          
            return 2
```

Reply:
Sure!